targets/csharp/make.js

```diff
@@ -1,10 +1,11 @@
+let ejs = require('ejs');
 var path = require("path");
 
 // Making resharper less noisy - These are defined in Generate.js
 if (typeof (copyOrTemplatizeFile) === "undefined") copyOrTemplatizeFile = function () { };
 if (typeof (templatizeTree) === "undefined") templatizeTree = function () { };
 if (typeof (generateApiSummaryLines) === "undefined") generateApiSummaryLines = function () { };
-if (typeof (getCompiledTemplate) === "undefined") getCompiledTemplate = function () { };
+//if (typeof (getCompiledTemplate) === "undefined") getCompiledTemplate = function () { };
```

Comment:
Is there any point in leaving getCompiledTemplate in Generate.js? Can it be removed?

Reply:
Not easily. The function is called by copyOrTemplatizeFile. It's still used for caching files for determining if we have "templatized" (remaining feature: visited) the file or not. What is returned is overwritten with our new changes from the make.js file